### PR TITLE
DevMay2016

### PR DESCRIPTION
**Pull request by ronaldddilley**
For the May Release code review.

### CODE REVIEW

File: processing/sensor.py [hunk 1]
```diff
@@ -108,15 +109,38 @@ def __init__(self, product_id):
...
+        self.sensor_name = SENSOR_INFO[self.sensor_code]['name']
+
+    @classmethod
+    def get_satellite_sensor_code(cls, product_id):
+        '''Returns the satellite-sensor code if known'''
+
+        old_prefixes = ['LT4', 'LT5', 'LE7',
+                        'LT8', 'LC8', 'LO8',
+                        'MOD', 'MYD']
+        collection_prefixes = ['LT04', 'LT05', 'LE07',
```
Comment (gschmidt-usgs) [inline processing/sensor.py]: I like this variable name.  I should have used the same.

File: processing/unittests/test_sensor.py [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+        method = sensor.is_lt08
+
+        self.assertTrue(method(self.lt08_product_id))
+        self.assertFalse(method(self.lo08_product_id))
+
+    def test_is_lo08(self):
+        method = sensor.is_lo08
+
+        self.assertTrue(method(self.lo08_product_id))
+        self.assertFalse(method(self.lt04_product_id))
```
Comment (gschmidt-usgs) [inline processing/unittests/test_sensor.py]: It's not totally obvious what LT04 had to do with LO08, but it appears there is some circular turn on / turn off going on with the individual sensors themselves.

**Comment by gschmidt-usgs** [comment]
We discussed an item internally via email to be considered in this release or a future release.  At this time I'm complete with the code review for this pull request.

File: processing/unittests/test_sensor.py (hunk 2 above)
Comment (ronaldddilley) [inline processing/unittests/test_sensor.py]: I just wanted to test at least one other product ID that would return False, and I did it in a circular pattern as I was adding the test methods.  But really each test method should check through all of the sensor options, and assert the correct response for each one.  I'll go through them and fix them up.

Comment (gschmidt-usgs) [inline processing/unittests/test_sensor.py]: I figured there was a method to your madness.

-----------------------------------------------------------------------------------------------------
Gail Schmidt
Software Engineer
Stinger Ghaffarian Technologies (SGT)
Contractor to the U.S. Geological Survey (USGS)
Earth Resources Observation and Science (EROS) Center
47914 252nd Street
Sioux Falls, SD 57198
Phone: 605-593-2272
gail.schmidt.ctr@usgs.gov

*NOTE – E-mail address change as of April 4, 2016.  Please update your
contact list accordingly!*

NOTICE:  This email may contain confidential, proprietary, or competition
sensitive bid or proposal procurement information.  Unauthorized disclosure
of this information may carry criminal penalties as set forth in the
Procurement Integrity Act, 41 U.S.C. 423, as amended.  Further, the
unauthorized disclosure of certain commercial information by civil servants
may result in fines or imprisonment under the Trade Secrets Act (18 U.S.C.
1905).  If you have received this information in error, please delete it,
including all copies, and notify the sender of the error immediately.

Comment (ronaldddilley) [inline processing/unittests/test_sensor.py]: Went through all of the is_<sensor> tests and added complete testing.

**Pull request merged**